src/examples/step2_test-deploy.yml

```diff
@@ -34,13 +34,30 @@ usage:
               tags:
                 only: /.*/
         # Because our publishing job has a tag filter, we must also apply a filter to each job it depends on.
+        - orb-tools/pack:
+            requires:
+              - command-tests
+              - <my-orb>/my_job
+            filters:
+              tags:
+                only: /.*/
+        - orb-tools/publish:
```

Comment:
I am not sure we need to bake in the dev orb publishing process by default. It may be good for documentation but I'm worried it could also complicate the example as it is not a required step.